lib/topological_inventory/openshift/collectors_pool.rb

```diff
@@ -0,0 +1,85 @@
+require "config"
+require "topological_inventory/openshift/collector"
+require "topological_inventory/openshift/logging"
+
+module TopologicalInventory::Openshift
+  class CollectorsPool
+    include Logging
+
+    def initialize(config_name, metrics, poll_time: 10)
+      self.collectors  = {}
+      self.collector_threads = {}
+      self.config_name = config_name
```

Comment:
This is probably a path not just a file name right?  I think `config_path` is more descriptive

Reply:
Oh I see it is actually just the filename, it'd give us more flexibility if we accepted a full path here that way we don't have to put the secrets file in a specific place, just gives us options.

Reply:
is it secure? It then loads any file from anywhere. Or do you consider it secure because it'll be used always in containers?

Reply:
Secure as in, making sure that no one passes an invalid file that could trigger an exploit?  Not sure how only reading files under a specific directory solves that.

Reply:
anyway, I think we don't need more flexibility for this, ConfigMap can be mounted to any place.